### ISSUE TITLE
tesseract-3.04.01 - .../bin/ld: cannot find -l-lOpenCL

### ISSUE THREAD

**Issue by devurandom** [tesseract-ocr/tesseract#386]
tesseract-3.04.01 does not build due to a bug in the `configure.ac` file:
```
libtool: link: x86_64-pc-linux-gnu-g++ -pipe -O2 -march=bdver3 -std=c++11 -Wl,-O1 -Wl,--hash-style=gnu -o .libs/tesseract tesseract-tesseractmain.o  -l-lOpenCL -Wl,--as-needed ./.libs/libtesseract.so -lrt -llept -lpthread
/usr/lib/gcc/x86_64-pc-linux-gnu/5.4.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -l-lOpenCL
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:591: tesseract] Error 1
make[2]: Leaving directory '/var/tmp/portage/app-text/tesseract-3.04.01/work/tesseract-3.04.01/api'
```

`tesseract-3.04.01-fix-opencl-ldflags.patch`:
```patch
--- tesseract-3.04.01/configure.ac.orig 2016-08-11 10:50:28.853292674 +0200
+++ tesseract-3.04.01/configure.ac      2016-08-11 11:06:32.919076995 +0200
@@ -248,7 +248,7 @@
         fi
         AC_SUBST([AM_CPPFLAGS], [-DUSE_OPENCL])
         OPENCL_CPPFLAGS="-I${OPENCL_INC}"
-        OPENCL_LDFLAGS="-l${OPENCL_LIBS}"
+        OPENCL_LDFLAGS="${OPENCL_LIBS}"
     fi
     ;;
 esac
```

See-Also: https://bugs.gentoo.org/show_bug.cgi?id=591070

**Comment by stweil** [comment]
Yes. And this bug was fixed in later versions. So please close this issue.

**Comment by devurandom** [comment]
It is not fixed on the 3.04 branch.
